mmv1/third_party/terraform/tests/resource_cloudids_endpoint_test.go

```diff
@@ -0,0 +1,95 @@
+package google
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestCloudIdsEndpoint_basic(t *testing.T) {
```

Comment:
I'm sorry I missed this before, but I was trying to find the VCR test result for this test. I think it isn't running because the function wasn't named "TestAcc"
```suggestion
func TestAccCloudIdsEndpoint_basic(t *testing.T) {
```

Reply:
Sure will do. Note that this test takes about 20 minutes to run locally. I'm slightly worried this will make VCR-test run time longer in the future.

Reply:
it should only be a problem when it replays in recording mode.

Seeing the tenancy issue, you can try the BootStrapSharedTestNetwork, but if it won't work, we can skip this in VCR (with skipIfVCR) for now (with a confirmation that it works in locally in your own project)

Reply:
Sure, I'll just re-run it both locally and on Github to make sure the current behaviour is consistent.

Reply:
Looks like the behaviour is consistent. Locally, if I set the resource to input:true, I'm also getting some unexpected behaviour. We can talk about this during our 1-1.